composer.json

```diff
@@ -12,7 +12,7 @@
         "php": "^7.3"
     },
     "require-dev": {
-        "phpspec/phpspec": "^7.0"
+        "phpspec/phpspec": "^7.4"
```

Comment:
I guess php version that should be changed not phpspec, right?

Reply:
Of course .. this is what happens when I get excited to make a contribution to an OP project lol

Reply:
No worries 😊